be1-go/channel/election/mod.go

```diff
@@ -105,8 +105,8 @@ type validVote struct {
 	// voteTime represents the time of the creation of the vote
 	voteTime int64
 
-	// indexes represents the indexes of the ballot options
-	indexes []int
```

Comment:
It is due to a change in the specifications.
We encrypted votes by encrypting the index inside the array, but that gave away information on the sender in the case of multiple choice questions. 
We cannot encrypt the array in itself at least with the current specification because it could grow arbitrarly in size and overflow the 64 bytes we have for the vote size.